test/k8sT/manifests.go

```diff
@@ -22,9 +22,11 @@ var (
 	deploymentManager = helpers.NewDeploymentManager()
 
 	DemoDaemonSet = helpers.Manifest{
-		Filename:     "demo_ds.yaml",
-		NumDaemonSet: 2,
-		NumPods:      1,
+		Filename:        "demo_ds.yaml",
+		DaemonSetNames:  []string{"testds", "testclient"},
```

Comment:
would be nice to have a helper which reads the file and fills `DaemonSetNames` and `DeploymentNames` automatically so we don't need to keep these in sync. I think it's a good candidate for a follow-up PR though.